ansibullbot/triagers/plugins/shipit.py

```diff
@@ -242,6 +242,13 @@ def get_shipit_facts(issuewrapper, meta, module_indexer, core_team=[], botnames=
             nmeta['owner_pr'] = True
             return nmeta
```

Comment:
This call is wrong here as well and could cause subsequent nmeta-changes to get lost, or make shipits fail to work.

Reply:
Fixed in #978